ArduCopter/mode_turtle.cpp

```diff
@@ -171,6 +171,11 @@ void ModeTurtle::output_to_motors()
 {
     // throttle needs to be raised
     if (is_zero(channel_throttle->norm_input_dz())) {
+        if (AP_HAL::millis() - last_throttle_warning_output_ms > 5000) {
```

Comment:
use const uint32_t now_ms = AP_HAL::millis()

Reply:
a shorter string is also needed for MissionPlanner

Reply:
Thanks - fixed